### PR TITLE
rename wikia-dev.com to wikia-dev.[us|pl]

### PR DESCRIPTION
**Pull request by hakubo**
## Links

* https://wikia-inc.atlassian.net/browse/XW-2627

## Description

Removing all occurencies of wikia-dev.com to remove later confusement.

### CODE REVIEW

File: server/config/settings.dev.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 export default {
 	loggers: {
 		console: 'debug'
 	},
-	devDomain: (process.env.WIKIA_DATACENTER === 'poz') ? 'pl' : 'us',
-	authCookieDomain: (process.env.WIKIA_DATACENTER === 'poz') ? '.wikia-dev.pl' : '.wikia-dev.us',
-	servicesDomain: (process.env.WIKIA_DATACENTER === 'poz') ? 'services.wikia-dev.pl' : 'services.wikia-dev.us',
+	devDomain,
+	authCookieDomain: `.wikia-dev.${devDomain}`,
+	servicesDomain,
```
Comment (rogatty) [inline server/config/settings.dev.js]: Let's be consistent and define it inline like `authCookieDomain`

**Comment by kvas-damian** [comment]
I think you can apply the same changes in https://github.com/Wikia/mobile-wiki ;)

File: front/main/mirage/config.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export default function () {
 	this.passthrough('/write-blanket-coverage');
 	this.passthrough('https://localhost/**');
 
-	this.passthrough('https://services-poz.wikia-dev.com/**');
+	this.passthrough('https://services-poz.wikia-dev.pl/**');
```
Comment (kvas-damian) [inline front/main/mirage/config.js]: It probably won't work on US devboxes :(

Comment (kvas-damian) [inline front/main/mirage/config.js]: BTW we don't use `services-poz` anymore.

**Pull request merged**